recipes/nmos-cpp/all/conanfile.py

```diff
@@ -73,57 +63,61 @@ def requirements(self):
 
     def build_requirements(self):
         # nmos-cpp needs CMake 3.17 or higher but CCI doesn't allow version ranges
-        self.build_requires("cmake/3.23.2")
+        self.build_requires("cmake/3.24.2")
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, 11)
+        if self.info.settings.compiler.get_safe("cppstd"):
+            build.check_min_cppstd(self, 11)
 
     def package_id(self):
         self.info.requires["boost"].minor_mode()
 
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
+        files.get(self, **self.conan_data["sources"][self.version],
+                  destination=self.source_folder, strip_root=True)
 
```

Comment:
@ericLemanissier suggestion:
```suggestion
        files.rm(self, "conanfile.txt", os.path.join(self.source_folder, "Development"))

```